.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] Summary of item"
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Thanks for reporting this! Please provide as much info as you can about the issue, so we can best help you. As a general guidance, pointers from https://stackoverflow.com/help/mcve can help you craft a good bug report.
+
+If this is a question please ask on [StackOverflow](https://stackoverflow.com/questions/tagged/consul).
```

Comment:
Maybe we should say "If this is a question _about the Consul API or Consul itself_"?

Reply:
done